packages/stack/src/views/Stack/Card.tsx

```diff
@@ -471,12 +475,9 @@ export default class Card extends React.Component<Props> {
 
     return (
       <View pointerEvents="box-none" {...rest}>
-        {overlayEnabled && overlayStyle ? (
-          <Animated.View
-            pointerEvents="none"
-            style={[styles.overlay, overlayStyle]}
-          />
-        ) : null}
+        {overlayEnabled && overlayStyle
+          ? this.props.overlay(overlayStyle)
```

Comment:
The code doesn't match the type definitions (should be `({ style: overlayStyle })`)